application/console/config/main.php

```diff
@@ -1,4 +1,10 @@
 <?php
+
+
+use Sil\PhpEnv\Env;
```

Comment:
I don't think we need this `use` statement anymore, since the env stuff was moved to the common config file.